File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/CLSKetQuaChiTietService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CLSKetQuaChiTietService : BaseService<CLSKetQuaChiTiet, ICLSKetQuaChiTietRepository>
    , ICLSKetQuaChiTietService
    {
    private readonly ICLSKetQuaChiTietRepository _CLSKetQuaChiTietRepository;
    public CLSKetQuaChiTietService(ICLSKetQuaChiTietRepository CLSKetQuaChiTietRepository) : base(CLSKetQuaChiTietRepository)
    {
    _CLSKetQuaChiTietRepository = CLSKetQuaChiTietRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/CLSKetQuaChiTiet.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CLSKetQuaChiTiet } from './CLSKetQuaChiTiet.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CLSKetQuaChiTietService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'CLSKetQuaChiTiet_Id', 'CLSKetQua_Id', 'DichVu_Id', 'DonViTinh', 'KetQua', 'KetQua2', 'MucBinhThuong', 'MucBinhThuongMin', 'MucBinhThuongMax', 'BatThuong', 'ThucHienBenNgoai', 'GioiHan'];

    List: CLSKetQuaChiTiet[] | undefined;
    ListFilter: CLSKetQuaChiTiet[] | undefined;
    FormData!: CLSKetQuaChiTiet;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CLSKetQuaChiTiet";
    }
}

